src/madpack/madpack.py

```diff
@@ -366,6 +366,8 @@ def _check_db_port(portid):
         row = _internal_run_query("SELECT version() AS version", True)
     except:
         error_(this, "Cannot validate DB platform type", True)
+    if portid == 'greenplum' and row[0]['version'].lower().find('greengage') >= 0:
```

Comment:
```suggestion
    version = row[0]['version'].lower()
    if portid == 'greenplum' and version.find('greengage') >= 0:
```
row[0]['version'].lower() is used 3 times now, IMHO it worst to simplify it already

Reply:
optimized